plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/utils/FullViewportAutoexposeHelper.java

```diff
@@ -0,0 +1,151 @@
+/* FeatureIDE - A Framework for Feature-Oriented Software Development
+ * Copyright (C) 2005-2017  FeatureIDE team, University of Magdeburg, Germany
+ *
+ * This file is part of FeatureIDE.
+ * 
+ * FeatureIDE is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * FeatureIDE is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with FeatureIDE.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * See http://featureide.cs.ovgu.de/ for further information.
+ */
+package de.ovgu.featureide.fm.ui.utils;
+
+import org.eclipse.draw2d.PositionConstants;
+import org.eclipse.draw2d.Viewport;
+import org.eclipse.draw2d.geometry.Insets;
+import org.eclipse.draw2d.geometry.Point;
+import org.eclipse.draw2d.geometry.Rectangle;
+import org.eclipse.gef.GraphicalEditPart;
+import org.eclipse.gef.editparts.ViewportAutoexposeHelper;
+
+/**
+ * Autoexposer which allows the pointer to be outside of the viewport
+ * 
+ * Refactered version of VIewportAutoexposeHelper. Does not implement AutoexposeHelper by itself because ViewportHelper is not visible.
+ */
+public class FullViewportAutoexposeHelper extends ViewportAutoexposeHelper {
+
+	/**
+	 * Simple stop watch
+	 * 
+	 * TODO: Is there any equivalent class in the APIs?
```

Comment:
If you only use the timer one time you can write the functionality directly into the code.